index.html

```diff
@@ -830,21 +830,26 @@ <h1><a name="2NormRefs">2 Normative references</a></h1>
 3-900-734-00-3.</p>
 
 <p class="NormRefDef"><a name="2-ICC-1">ICC-1</a>, International
-Color Consortium, "Specification ICC.1: 1998-09, File Format for
-Color Profiles", 1998, available at <code><a href=
-"http://www.color.org/">http://www.color.org/</a></code></p>
-
-<p class="NormRefDef"><a name="2-ICC-1A">ICC-1A</a>,
-International Color Consortium, "Specification ICC.1A: 1999-04,
-Addendum 2 to ICC.1: 1998-09", 1999, available at <code><a href=
-"http://www.color.org/">http://www.color.org/</a></code></p>
+Color Consortium, "Specification ICC.1:2010-12 (Profile version 4.3.0.0)
+Image technology colour management - Architecture, profile format, and data structure,
+available at <code><a href=
+"http://www.color.org/icc_specs2.xalter">http://www.color.org/icc_specs2.xalter</a></code>
+also published as ISO 15076-1</p>
+
+<p class="NormRefDef"><a name="2-ICC-a">ICC-2</a>,
```

Comment:
The link target changed from 2-ICC-1A to 2-ICC-a.

The places that point to this fragment should be updated as well.